plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -1013,5 +1014,53 @@ describe('pg', () => {
         done();
       });
     });
+
+    it('should generate db.client.operation.duration metric with error attribute', done => {
+      client.query('SELECT foo from bar', async (err, ret) => {
+        assert.notEqual(err, null);
+        const { resourceMetrics, errors } = await metricReader.collect();
+        assert.deepEqual(
+          errors,
+          [],
+          'expected no errors from the callback during metric collection'
+        );
+
+        const metrics = resourceMetrics.scopeMetrics[0].metrics;
+        assert.strictEqual(
+          metrics[0].descriptor.name,
+          METRIC_DB_CLIENT_OPERATION_DURATION
+        );
+        assert.strictEqual(
+          metrics[0].descriptor.description,
+          'Duration of database client operations.'
+        );
+        const dataPoint = metrics[0].dataPoints[0];
+        assert.strictEqual(
+          dataPoint.attributes[SEMATTRS_DB_SYSTEM],
+          DBSYSTEMVALUES_POSTGRESQL
+        );
+        // TODO: add once #2475 is merged
```

Comment:
It's approved :)

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2475 is now merged too :slightly_smiling_face:

Reply:
Merged with main and added the code